examples/gno.land/r/demo/profile/profile.gno

```diff
@@ -53,42 +53,49 @@ var boolFields = map[string]bool{
 
 // Setters
 
-func SetStringField(field, value string) error {
+func SetStringField(field, value string) bool {
 	addr := std.PrevRealm().Addr()
-	if _, ok := stringFields[field]; !ok {
-		return errors.New("invalid string field")
+	key := addr.String() + ":" + field
+	updated := fields.Set(key, value)
+
+	event := "ProfileFieldCreated"
```

Comment:
These event keys should be constants, along with some keys

Reply:
thanks @zivkovicmilos  for your review. I've updated the code including some minor refactoring to use constants instead of hardcoded strings for other places